docker/buildDocker.sh

```diff
@@ -99,51 +99,46 @@ checkArgs()
         	echo "WORKSPACE not found, setting it as environment variable 'HOME'"
         	WORKSPACE=$HOME
 	fi
-	if [[ "$useEclipseDockerFiles" == true && "$useEclipseDockerSlavesFiles" == true ]]; then
-		echo "Unable to use both kinds of dockerfiles at once, Select a single option."
-		exit 1
-        fi
+	if [ "$cleanWorkspace" == true ]; then
+		echo "Cleaning workspace"
+		rm -rf $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant
+	fi
 }
 
 useEclipseDockerFiles()
```

Comment:
I wonder if that needs to be called eclipseOpenJ9DockerFiles throughout the script.  I know it seems overkill, but I thought initially that Eclipse had some special docker base-files for all Java  variants (as opposed to OpenJ9)

Reply:
Happy to make the change, if it can be clearer, it should be :-)